src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IEventReferenceOperation.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void IEventReference_AddEvent_StaticEventWithInstanceReceiver()
```

Comment:
>IEventReference_AddEvent_StaticEventWithInstanceReceiver [](start = 20, length = 56)

Please make sure we have tests for accessing an event (static and instance) of off a type

Reply:
Done.

---
In reply to: [149248586](https://github.com/dotnet/roslyn/pull/22972#discussion_r149248586) [](ancestors = 149248586)